fireapp/lib/presentation/volunteer_information/volunteer_information_viewmodel.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:fireapp/domain/models/volunteer_information.dart';
+import 'package:fireapp/domain/repository/volunteer_information_repository.dart';
+import 'package:fireapp/domain/request_state.dart';
+import 'package:fireapp/global/di.dart';
+import 'package:fireapp/presentation/fireapp_view_model.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:injectable/injectable.dart';
+import 'package:rxdart/rxdart.dart';
+
+@injectable
+class VolunteerInformationViewModel extends FireAppViewModel{
+  final VolunteerInformationRepository volunteerInformationRepository;
+
+  final BehaviorSubject<RequestState<VolunteerInformation>>
+    _volunteerInformation = BehaviorSubject.seeded(RequestState.initial());
+  Stream<RequestState<VolunteerInformation>> get volunteerInformation =>
+      _volunteerInformation.stream;
+
+  VolunteerInformationViewModel(this.volunteerInformationRepository);
+
+  void getVolunteerInformation(String volunteerId) {
+    () async {
+      _volunteerInformation.add(RequestState.loading());
+      try {
+        _volunteerInformation.add(RequestState.success
+          (await volunteerInformationRepository.getVolunteerInformation(volunteerId)));
+      } catch (e) {
+        logger.e(e);
+        _volunteerInformation.add(RequestState.exception(e));
+    }
+    }();
+  }
+
+  String formatHours(List<List<int>> hours) {
```

Comment:
Why do you declare these functions in both the view and viewmodel? Pick one and get rid of the other.